Server/mods/deathmatch/acl.xml

```diff
@@ -29,7 +29,7 @@
       <acl name="SuperModerator" />
       <acl name="Admin" />
       <acl name="RPC" />
-      <object name="user.Console" />
```

Comment:
Why? "Console" is build in account that you can not login, it is used just for Console